game-landing.css

```
@@ -0,0 +1,306 @@
+   - Poo-zzle keeps the default glRunAndDie above, untouched.
+   - 5 Star Game: trips face-first at the middle of the screen,
+     lies there a beat, picks himself up and jogs off the end.
+   - Cheetah Crawl: throws a rope to the top of the screen and
+     Tarzan-swings up and out of view, then does it all again.
+   ============================================================ */
+
+/* ---------- 5 Star Game — the mid-screen faceplant ---------- */
+.gl-stickman--fall{
+  animation:glTripFall 6s ease-in-out infinite;
+}
+@keyframes glTripFall{
```

Stylelint (17.14.0)

[error] Expected keyframe name "glTripFall" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)